artifactory.py

```diff
@@ -385,6 +389,7 @@ def quote_url(url):
     :param url: (str) URL that should be quoted
     :return: (str) quoted URL
     """
+    logging.debug(f"Raw URL passed for encoding: {url}")
```

Comment:
Do we need these logging? Look like some one forgot them after debugging in the console.

Standard `request` logs should be enough for figuring out the problem in the result URL. 

If we really want to log it - let's create `quote_url_and_log`

Why do we use "percent-encoded url" as reference here, if we are inside "quote_url" function? It should be either `Quoted URL:` in log or `encode_url` function
```python
def encode_url_and_log(url):
    logging.debug(f"Raw URL: {url}")
    url = quote_url(url)
    logging.debug(f"Encoded URL: {quoted_url}")
    return quoted_url
```

Reply:
I do not know why this name was selected for the function, but this is called "percent encoding" in HTTP

I agree that request will handle it, let's use only Raw URL. We already faced couple of bugs with this encoding, thus, would be good to log